### PR TITLE
assigment 4

### PR DESCRIPTION
**Pull request by Miguel22hp**

### CODE REVIEW

File: Assignment4/Miguel22hp-c200113/task07.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+""":**TASK 7.1: List all subclasses of "LivingThing" with RDFLib and SPARQL**"""
+
+# TO DO
+from rdflib.plugins.sparql import prepareQuery
+
+q1 = prepareQuery("""
+    SELECT ?subClass
+    WHERE {
+        ?subClass rdfs:subClassOf ns:LivingThing .
```
Comment (dgarijo) [inline Assignment4/Miguel22hp-c200113/task07.py:32]: This won't retrieve all subclasses of subclasses of Living Thing. You need to use `+`

File: Assignment4/Miguel22hp-c200113/task07.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+"""
+
+# TO DO
+query = prepareQuery("""
+    SELECT ?individual ?property ?value ?class_
+    WHERE {
+        ?individual a ?class_ .
+        ?individual ?property ?value .
+        FILTER (?class_ = ns:Person || ?class_ = ns:Animal)
```
Comment (dgarijo) [inline Assignment4/Miguel22hp-c200113/task07.py:66]: this is a very inefficient query. Instead, use UNION

File: Assignment4/Miguel22hp-c200113/task07.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+# Visualize the results
+for row in g.query(query):
+    print(row.name)
+
+"""**Task 7.5: List the entities who know at least two other entities in the graph**
+
+> Bloque con sangría\
+
+
+"""
```
Comment (dgarijo) [inline Assignment4/Miguel22hp-c200113/task07.py:106]: Missing 7.5.
missing exercises with rdflib

**Review by dgarijo — CHANGES_REQUESTED** [review]

**Comment by anaigmo** [comment]
Entregado con errores sin corregir, no se aceptan más PR

**Pull request merged**